### DISCUSSION TITLE
Is it possible to add PotPlayer to the supported app?

### DISCUSSION THREAD

**Discussion by fadiajlil** [mach64/trakt-scrobbler#227] [Ideas]
PotPlayer is a great media player with a lot of customization, So it would be great if it can work with Trakt-Scobbler.

**Comment by iamkroot** [comment]
See #203. In short, it is not possible since potplayer does not provide any programmatic way to query the currently playing media.

**Comment by fadiajlil** [comment]
I have been using unified remote with potplayer and it was giving the title of the media playing, and if the media if organized and named correctly isn't that enough info for trakt.
I'm not a coder but I found this: 
"  actions.update = function ()
	  local hwnd = win.find("PotPlayer", nil);
	  if (hwnd == 0) then
		  hwnd = win.find("PotPlayer64", nil);
	  end
	  local _title = win.title(hwnd);
	  _title = utf8.replace(_title, " - Daum PotPlayer", "");
	  if (_title == "") then
		  _title = "[Not Playing]";
	  end
	
	  if (_title ~= title) then
		  title = _title;
		  server.update({ id = "info", text = title });
	  end
  end  "
I think this what they are using to get the info from the app.

**Reply by iamkroot** [reply]
Yep I've seen this (Unified Remote is awesome!) But as you can see, the info is only limited to title. We also need the duration and progress.

**Reply by fadiajlil** [reply]
that's a shame I hope some major update make it more friendly to programmers like vlc.
thank you for your hard work it's much appreciated.

**Comment by AlesRFK** [comment]
I'm using Manic Time, that could scrobb any data from any player.

**Reply by iamkroot** [reply]
Could you post a screenshot of what data it extracts? Can it extract the current play elapsed time, total duration, etc.?

**Reply by AlesRFK** [reply]
![image](https://github.com/user-attachments/assets/8885b030-40fe-4871-a441-6bec34cddbc2)
Sorry for the delay. I thought about it and maybe it's too much work for nothing. MT scrape data from the window/process title and measure time spend in the app.
So you would need to add progress in the PP like "Movie 0-100%" anyway wich is mentioned above OR scrape data from Total Time (too complicated).
Do you have any updates for the PotPlayer extension?
